packages/dd-trace/src/config.js

```diff
@@ -644,49 +273,10 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       }
     }
 
-    this.remoteConfig = {
-      enabled: DD_REMOTE_CONFIGURATION_ENABLED,
-      pollInterval: DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS
-    }
-    this.iast = {
-      enabled: isTrue(DD_IAST_ENABLED),
-      requestSampling: DD_IAST_REQUEST_SAMPLING,
-      maxConcurrentRequests: DD_IAST_MAX_CONCURRENT_REQUESTS,
-      maxContextOperations: DD_IAST_MAX_CONTEXT_OPERATIONS,
-      deduplicationEnabled: DD_IAST_DEDUPLICATION_ENABLED,
-      redactionEnabled: DD_IAST_REDACTION_ENABLED,
-      redactionNamePattern: DD_IAST_REDACTION_NAME_PATTERN,
-      redactionValuePattern: DD_IAST_REDACTION_VALUE_PATTERN,
-      telemetryVerbosity: DD_IAST_TELEMETRY_VERBOSITY
-    }
-
-    this.isCiVisibility = isTrue(DD_IS_CIVISIBILITY)
-
-    this.isIntelligentTestRunnerEnabled = this.isCiVisibility && isTrue(DD_CIVISIBILITY_ITR_ENABLED)
-    this.isGitUploadEnabled = this.isCiVisibility &&
-      (this.isIntelligentTestRunnerEnabled && !isFalse(DD_CIVISIBILITY_GIT_UPLOAD_ENABLED))
-
-    this.gitMetadataEnabled = isTrue(DD_TRACE_GIT_METADATA_ENABLED)
-    this.isManualApiEnabled = this.isCiVisibility && isTrue(DD_CIVISIBILITY_MANUAL_API_ENABLED)
-    this.isEarlyFlakeDetectionEnabled = this.isCiVisibility && isTrue(DD_CIVISIBILITY_EARLY_FLAKE_DETECTION_ENABLED)
-
-    this.openaiSpanCharLimit = DD_OPENAI_SPAN_CHAR_LIMIT
-
     // Requires an accompanying DD_APM_OBFUSCATION_MEMCACHED_KEEP_COMMAND=true in the agent
     this.memcachedCommandEnabled = isTrue(DD_TRACE_MEMCACHED_COMMAND_ENABLED)
-
-    this.stats = {
-      enabled: isTrue(DD_TRACE_STATS_COMPUTATION_ENABLED)
-    }
-
-    this.traceId128BitGenerationEnabled = isTrue(DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED)
-    this.traceId128BitLoggingEnabled = isTrue(DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED)
-
-    this.isGCPFunction = isGCPFunction
     this.isAzureFunctionConsumptionPlan = isAzureFunctionConsumptionPlan
```

Comment:
instead of getting it in line 145, so far from here
```suggestion
    this.isAzureFunctionConsumptionPlan = getIsAzureFunctionConsumptionPlan()
```